tests/test_existing_excerpt.py

```diff
@@ -0,0 +1,93 @@
+import pytest
+import time
+import requests
+from selenium import webdriver
+from selenium.webdriver.common.by import By
+from selenium.common.exceptions import NoSuchElementException
+from selenium.webdriver.common.keys import Keys
+
+@pytest.fixture
+def prerequisite(base_url, selenium):
+    # create new excerpt only for this testing
+    selenium.get('{0}/admin/login/'.format(base_url))
+    selenium.maximize_window()
+
+    # login as admin
+    username = selenium.find_element_by_id('id_username')
+    password = selenium.find_element_by_id('id_password')
+    login = selenium.find_element_by_class_name('submit-row')
+    username.send_keys("admin")
+    password.send_keys("admin")
+    login.send_keys(Keys.RETURN)
+
+    selenium.get('{0}/'.format(base_url))
+
```

Comment:
The code to create a new excerpt is shared between the preparation of this test and the actual test procedure of the "new excerpt" test. Extract it into a function and call it from both places.